### PR TITLE
test: fix host vuln test

### PR DESCRIPTION
**Pull request by dmurray-lacework**
fix logic in host vuln test

### CODE REVIEW

File: integration/host_vulnerability_test.go [hunk 1]
```diff
@@ -414,7 +414,10 @@ func validRFC3339(field string, jsonString string) bool {
 
 func machineIDExists(lw *api.Client) bool {
 	resp, err := lw.Vulnerabilities.Host.GetHostAssessment(machineID)
-	return err != nil || !resp.Ok
+	if err != nil {
```
Comment (rmoles) [inline integration/host_vulnerability_test.go:417]: Wondering if it could be reduced to `return err == nil && resp.Ok` ?

Comment (dmurray-lacework) [reply integration/host_vulnerability_test.go:417]: Could do, but I think the way it is now is a little more clear. We had a 1 liner here previously.

Comment (dmurray-lacework) [reply integration/host_vulnerability_test.go:417]: Gonna leave as it for now

**Review by rmoles — APPROVED** [review]
LGTM

**Pull request merged**